explanation/dnssec/dnssec.md

```diff
@@ -0,0 +1,254 @@
+(dnssec)=
+# DNS Security Extensions (DNSSEC)
+
+DNS is a mapping system between names and IP addresses. Besides this main purpose that allows us to use friendly names instead of a sequence of numbers to reach a web site, it may store more information about a particular domain, such as:
```

Comment:
It makes sense to talk about about DNS first, but I think the first sentence should at least reference, if not defin\
e, DNSSEC.  For example:

    DNSSEC is a security extension for the Doman Name System (DNS).  DNS is a mapping between names and Internet Protocol (IP) addresses, allowing the use of friendly names instead of sequences of numbers when reaching web sites.  Additionally, it stores extra information about a given domain, such as:

I wordsmithed that first paragraph a bit.

For your list I notice several lines start with "when" or "what".  For consistency use that phrasing on the other lines.  E.g. "who is the point of contact", "how host names are mapped to IP addresses, and vice-versa", etc.

Reply:
great suggestions